Index: src/use_case/create_playlist/CreatePlaylistOutputBoundary.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package use_case.create_playlist;\n\npublic interface CreatePlaylistOutputBoundary {\n}\n
===================================================================
diff --git a/src/use_case/create_playlist/CreatePlaylistOutputBoundary.java b/src/use_case/create_playlist/CreatePlaylistOutputBoundary.java
--- a/src/use_case/create_playlist/CreatePlaylistOutputBoundary.java	
+++ b/src/use_case/create_playlist/CreatePlaylistOutputBoundary.java	
@@ -1,4 +1,5 @@
 package use_case.create_playlist;
 
 public interface CreatePlaylistOutputBoundary {
+
 }
